### PR TITLE
[CBRD-24636] add new testcase for Result error in join query using USE_DESC_IDX hint

### PR DESCRIPTION
**Pull request by tw-kang**
http://jira.cubrid.org/browse/CBRD-24636

### CODE REVIEW

File: sql/_13_issues/_23_1h/cases/cbrd_24636.sql [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+-- use no_multi_range_opt hint.
+select /*+ recompile ordered no_multi_range_opt */ t.y
+from t_a inner join t_b as t 
+on t_a.x = t.x 
+and t_a.x = 1
+where t.x = 1
+order by t.y
+limit 1;
+
+drop t_a, t_b;
```
Comment (youngjinj) [inline sql/_13_issues/_23_1h/cases/cbrd_24636.sql]: This should be done in another issue.
Please exclude it here.

File: sql/_13_issues/_23_1h/answers/cbrd_24636.answer [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+                            index: idx_desc term[?]
+                            sargs: term[?]
+                            cost:  ? card ?
+                 cost:  ? card ?
+    sort:  ? asc
+    cost:  ? card ?
+Query stmt:
+select /*+ ORDERED NO_MULTI_RANGE_OPT */ t.y from t_a t_a inner join t_b t on t_a.x=t.x and t_a.x= ?:?  and ?=? where t.x= ?:?  order by ? for orderby_num()<= ?:? 
+===================================================
+0
```
Comment (youngjinj) [inline sql/_13_issues/_23_1h/answers/cbrd_24636.answer]: This should be done in another issue.
Please exclude it here.

File: sql/_13_issues/_23_1h/cases/cbrd_24636.sql (hunk 1 above)
Comment (tw-kang) [reply sql/_13_issues/_23_1h/cases/cbrd_24636.sql]: I removed the contents from the test case.

File: sql/_13_issues/_23_1h/answers/cbrd_24636.answer (hunk 2 above)
Comment (tw-kang) [reply sql/_13_issues/_23_1h/answers/cbrd_24636.answer]: I removed the contents from the test case.

**Review by kwonhoil — APPROVED** [review]

**Review by ssihil — APPROVED** [review]

**Review by youngjinj — APPROVED** [review]

**Pull request merged**